### ISSUE TITLE
[🐞] Is this project dead?

### ISSUE THREAD

**Issue by AwakenTheJaken** [BuilderIO/partytown#510]
### Describe the bug

There has been no updates or any contributors popping in to help with issues. Firefox for one... Our dev team is pretty disappointed as we spent many hours implementing and testing this, Firefox is rendered unusable on our site, but wait... there's more!

### Reproduction

http://localhost:3000

### Steps to reproduce

Using the library

### Browser Info

Chrome

### Additional Information

Can we get a absolute if this project has been archived? Also there are many many bugs, Adding this to our application had a huge impact on our numbers on GA4, CSP is implemented correctly yet things are falling through the cracks.  Our team has since removed this from our stack due to the issues above.

**Comment by mhevery** [comment]
There are several reasons why there have not been many updates.

1. people file issues without reproduction. 
2. The nature of this project is that any reproduction often involves debugging 3rd-party minified code, which is extremely hard and time-consuming.
3. We have been putting most of our efforts into Qwik.

Our hope is that as we grow and gain more resources, we should be able to do a better job supporting it.

Have you tried to create a PR to fix an issue?

**Comment by emish89** [comment]
I understand your points but I can totally understand the user that opened this.

The firefox bug is really making the library useless and can be a big problem for who implemented it and now needs to migrate until resolution.
...and I think there are a lot of example of codes to debug it. 

I don't know the effort required but can really help to face this issue with an high priority, because firefox is widely used and the error cannot be bypassed

**Comment by thejackshelton** [comment]
> I understand your points but I can totally understand the user that opened this.
> 
> The firefox bug is really making the library useless and can be a big problem for who implemented it and now needs to migrate until resolution. ...and I think there are a lot of example of codes to debug it.
> 
> I don't know the effort required but can really help to face this issue with an high priority, because firefox is widely used and the error cannot be bypassed

Agreed that this is a big problem. Happy to look into a PR someone makes to fix it.

**Comment by emish89** [comment]
https://github.com/BuilderIO/partytown/pull/511

can you please review this? I don't know why the setTimeouts are needed but without them seems working on firefox and chrome. 

@thejackshelton @mhevery

**Comment by mhevery** [comment]
> #511
> 
> can you please review this? I don't know why the setTimeouts are needed but without them seems working on firefox and chrome.

Oh, that is interesting and exciting! Any chance you could add a test to https://github.com/BuilderIO/partytown/tree/main/tests/platform which shows that the issue has been fixed?

**Comment by gioboa** [comment]
https://github.com/BuilderIO/partytown/pull/511 close this one
